package.json

```diff
@@ -53,7 +53,7 @@
 		"husky": "^4.3.0",
 		
 		"jasmine": "^3.6.4",
-		"jasmine-core": "^3.4.0",
+		"jasmine-core": "3.4.0",
```

Comment:
This was needed to resolve some FireFox test failures. I've opened #742 to look into this